tests/test_packages/test_packages.py

```diff
@@ -39,6 +39,15 @@ def test_album_isInList(self):
                         response.status_code)
 
 
+    def test_manifesting_inList(self):
+        """ Need to test OTHER_MIRRORS capailities. tm-manifesting should typically
+        come from OTHER_MIRRORS, so check if it is in the list of all packages.
+        NOTE: This might not be true at some point.
+        """
+        response = self.client.get(self.base_endpoint + 'package/tm-manifesting')
+        self.assertTrue(response.status_code == 200)
```

Comment:
WHat is the benefit of this instance method over a native assertion?

Reply:
Error and success reports to the console, of what I understand.
If you run the test, it will show you something like:
3/3 tests passed...

Reply:
Oh, this is a testing module, not "production" source.   My bad.